instance/models/openedx_instance.py

```diff
@@ -77,7 +77,7 @@ class OpenEdXInstance(
     # Most settings/fields are inherited from mixins
 
     successfully_provisioned = models.BooleanField(default=False)
-    database_name = models.CharField(blank=True, max_length=60, default='')
+    database_name = models.CharField(blank=False, unique=True, max_length=60, default='')
```

Comment:
Nit: we can remove the `default=''` now that every new instance is required to have a unique database_name.